network/src/protocols/mod.rs

```diff
@@ -167,7 +167,7 @@ impl CKBProtocol {
             })
             .support_versions(supported_versions)
             .service_handle(move || {
-                ProtocolHandle::Both(Box::new(CKBHandler {
+                ProtocolHandle::Callback(Box::new(CKBHandler {
```

Comment:
I wonder which behaviour changed after modifying `ProtocolHandle::Both` to `ProtocolHandle::Callback`?

Reply:
Yes, please check @driftluo 's review comment, it should use `Callback` mode here.

Reply:
We need some documents about Both and Callback